lib/omnibus/software.rb

```diff
@@ -52,6 +52,10 @@ def load(project, name, manifest)
         end
       end
 
+      def reset!
```

Comment:
Need a yard comment here, see the following for a good example:
https://github.com/chef/omnibus/blob/798b2db8e34a296d074006086288cfb6693c9d65/lib/omnibus/config.rb#L76-L81